File: server/src/main/java/com/gyfish/formflow/config/AppResponse.java
package com.gyfish.formflow.config;

import lombok.Data;

@Data
public class AppResponse {

    private int code;
    private String msg;
    private Object data;

    AppResponse(int code, String msg, Object data) {
        this.code = code;
        this.msg = msg;
        this.data = data;
    }

    public static AppResponse ok(Object data) {
        return new AppResponse(0, "ok", data);
    }

}

File: service-form/src/util/AppResponse.ts
export class AppResponse {
  //
  code: number
  msg: string
  data: any

  constructor(c: number, m: string, d: any) {
    this.code = c
    this.msg = m
    this.data = d
  }

  static ok(data: any): AppResponse {
    return new AppResponse(0, 'ok', data)
  }
}
